### DISCUSSION TITLE
Do i need to merge the new branch ?

### DISCUSSION THREAD

**Discussion by ajitsingh09** [open-sauced-craftwork/browser-extension#53] [Q&A]
<img width="683" alt="Screenshot 2023-03-31 022602" src="https://user-images.githubusercontent.com/111536019/229276421-c9c15b9e-d41f-46ba-bb69-11402dac2ac9.png">

**Comment by anush008** [comment]
You need to wait for @bdougie to review your submission. I recommend you to ping him once in the PR.
